{"name": "test_create_order_with_invalid_ingredient", "status": "failed", "statusDetails": {"message": "AssertionError: assert 400 == 500\n +  where 400 = <Response [400]>.status_code", "trace": "self = <test_create_order.TestCreateOrder object at 0x1035cc9b0>\nauth_token = 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjY3NDdjZWRjOWVkMjgwMDAxYjUyNjdlNCIsImlhdCI6MTczMjc2NTcxNywiZXhwIjoxNzMyNzY2OTE3fQ.wkW4KWVTxzU2MffwPsxn2t-UfrG0EoAF2WGkJFcF9Fc'\n\n    @allure.feature(\"Создание заказа\")\n    @allure.story(\"Попытка создать заказ с невалидными ингредиентами\")\n    def test_create_order_with_invalid_ingredient(self, auth_token):\n        order_data = {\"ingredients\": [\"invalid_hash\"]}\n        headers = {\"Authorization\": f\"{auth_token}\"}\n        response = requests.post(f\"{BASE_URL}/orders\", json=order_data, headers=headers)\n>       assert response.status_code == 500\nE       assert 400 == 500\nE        +  where 400 = <Response [400]>.status_code\n\ntest_create_order.py:38: AssertionError"}, "start": 1732765717392, "stop": 1732765717693, "uuid": "ee3272d0-5359-4234-a554-bf01b7894024", "historyId": "9bf7d65d22acbc46bdc4ddb4293ba878", "testCaseId": "9bf7d65d22acbc46bdc4ddb4293ba878", "fullName": "tests.test_create_order.TestCreateOrder#test_create_order_with_invalid_ingredient", "labels": [{"name": "feature", "value": "Создание заказа"}, {"name": "story", "value": "Попытка создать заказ с невалидными ингредиентами"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_order"}, {"name": "subSuite", "value": "TestCreateOrder"}, {"name": "host", "value": "admins-MacBook-Pro.local"}, {"name": "thread", "value": "25889-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_order"}]}